bigquery/api/sync_query.py

```diff
@@ -66,7 +66,7 @@ def main(project_id, query, timeout, num_retries):
             pageToken=page_token,
             **query_job['jobReference']).execute(num_retries=2)
 
-        print(json.dumps(page['rows']))
+        print(json.dumps(page.get('rows', [])))
```

Comment:
Not a fan. The appeal of this script is its composability through valid JSON output on stdout; you can pipe it to `jq` or other JSON tools. Incidentally this is also why I chose for []; by always returning an array you can just loop over the result no matter what.